### PR TITLE
[Vm-repair] Fix in az vm repair run --preview parameter

### PR DESCRIPTION
**Pull request by haagha**
---

(https://github.com/Azure/azure-cli-extensions/issues/6820)
Fixing Run script to allow variable fork and branch names.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Vm-repair

**Review by zhoxing-ms — APPROVED** [review]

File: src/vm-repair/HISTORY.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 
 Release History
 ===============
+
+0.5.8
+++++++
+Fixing az vm repair run --preview parameter to take in fork and branch name of User's repository.
```
Comment (zhoxing-ms) [inline src/vm-repair/HISTORY.rst:7]:
```suggestion
Fix az vm repair run --preview parameter to take in fork and branch name of User's repository.
```

**Comment by wangzelin007** [comment]
> > Vm-repair
> 
> Hi guys, can you add me as a reviewer as well? @yonzhan @zhoxing-ms

Do you mean to add you to the reviewer list only for this PR, or will you be responsible for the review of all vm repair related PRs?

**Comment by haagha** [comment]
> > > Vm-repair
> > 
> > 
> > Hi guys, can you add me as a reviewer as well? @yonzhan @zhoxing-ms
> 
> Do you mean to add you to the reviewer list only for this PR, or will you be responsible for the review of all vm repair related PRs?

@qwuae1 will be owning this extension that I previously used to own so all vm repair related PR's.

@wangzelin007 can you merge this. Thanks.

**Pull request merged**